packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.Props.ts

```diff
@@ -29,4 +31,13 @@ export interface IResizeGroupProps extends React.HTMLAttributes<ResizeGroup | HT
    * an infinite render loop.
   */
   onReduceData: (prevData: any) => any;
+
+  /**
+   * Function to be called when the measuring pass is complete and renderedData is shown on screen.
+   * It is only called when the data used to render does not match that of the last render.
```

Comment:
My proposal was to invoke the callback whenever we render the real thing, not only if the data is different than the previous one. Basically remove this line: this.state.renderedData !== this._lastRenderedData and have the consumer implement that. This way there are no assumptions about whether the data is mutable or immutable.